### ISSUE TITLE
composer bootstrap v4 really needs PhpBrowser?

### ISSUE THREAD

**Issue by zdenekgebauer** [Codeception/Codeception#5806]
#### What are you trying to achieve?
Boostrap new tests without unused modules

#### What do you get instead?
```
In ModuleContainer.php line 110:
                                                        
  Module PhpBrowser is not installed.                   
  Use Composer to install corresponding package:        
                                                        
  composer require codeception/module-phpbrowser --dev  
```

#### Steps to reproduce:
1. create new composer. json  with 
```
  "require-dev": {
    "codeception/codeception": "^4.0",
    "codeception/module-asserts": "^1.1"
  }
```
2. run ```composer install```
3. run ```vendor\bin\codecept bootsrap```

### Details

* Codeception version:
* PHP Version: 7.3
* Operating System: Windows 10
* Installation type: Composer
* List of installed packages (`composer show`)
* Suite configuration: 

Presence of PhpBrowser shouldn't be checked during boostrap. Some projects do not require acceptance nor functional tests, they require only unit test

**Comment by Naktibalda** [comment]
@zdenekgebauer If you only want unit suite, use `codecept init unit` instead of `codecept bootstrap`.

Though `bootstrap` really requires some testing and bug fixing.

**Comment by zdenekgebauer** [comment]
Yes, this is little confusing:  
- module Db is not required on bootstrap. Error about missing module appears after adding module to .yml and running test
- module Asserts is also required on bootstrap

**Comment by Naktibalda** [comment]
Two issues here.
1. Modules should be installed before creating suites: https://github.com/Codeception/Codeception/blob/4.0.1/src/Codeception/Template/Bootstrap.php#L42-L48
2. Even if the modules are installed before creating suite (`codecept init unit` does it in the right order), [createHelper method](https://github.com/Codeception/Codeception/blob/4.0.1/src/Codeception/Template/Bootstrap.php#L42-L48) tries to load newly created Helper class, and the error that you got above happens, because modules weren't installed when vendor/autoload.php file  was loaded. I will try to reload it.

**Comment by Naktibalda** [comment]
Actually `createHelper` is fine, it is `createActor` which breaks bootstrap.

**Comment by Naktibalda** [comment]
I released patch as 4.0.2
